Tools/CISupport/ews-build/events.py

```diff
@@ -313,14 +313,19 @@ def stopService(self):
 
 class GitHubEventHandlerNoEdits(GitHubEventHandler):
     ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
+    OPEN_STATES = ('open',)
```

Comment:
Just for my understanding, what are all possible states?

Reply:
According to the documentation, just `open` and `closed`. `draft` is a separate variable.